### DISCUSSION TITLE
How to turn a reference image to another style in Fooocus?

### DISCUSSION THREAD

**Discussion by gin063** [lllyasviel/Fooocus#2625] [Q&A]
First of all, thanks to the Fooocus team for developing a powerful tool, I have a question that I need help with: how to turn a reference image to another style in Fooocus? Such as turn a real photo to oil painting style.
I still can't figure it out after trying many times, it always generate an image in the same style of the uploaded image.
It's easy to do with Midjourney, for example, I just need to upload my image then type an art style and the -iw parameter.
My steps in Fooocus:
1. upload my image in image prompt window
2. select a specific art style
3. write a simple prompt
model: Juggernaut XL v8
version: 2.3.1
Should I change any parameters in advanced settings?

**Comment by Rogala** [comment]
[FaceSwap/Pyracanny/CPDS & More Explained](https://www.youtube.com/watch?v=0vzunoCYiMI)

**Reply by gin063** [reply]
Thank you.

**Comment by ILikeToasters** [comment]
This might give you some ideas. It is showing how to do anime but can work with others. Make sure you disable the default styles and choose the ones geared towards what you are trying to get. https://youtu.be/PuC2AVrQXTs

**Reply by gin063** [reply]
Thanks, it's helpful.

**Comment by Umithubble** [comment]
explanations are insufficient. The image is either anime or a very realistic photograph. I want to start an art movement called "**anti anime**".
